### PR TITLE
[RFC] Add Features Lead Handbook

### PR DESCRIPTION
**Pull request by justaugustus**
Documenting the Features Lead process.
WIP, but I'm getting this out of a [Google Doc](https://docs.google.com/document/d/1WPB0N4kjVN_CY205dBEzs9KULWgkBeW1GASV4YHaK_w/edit?usp=sharing) and up for review. :)

Closes: #124 

/sig release pm
/cc @idvoretskyi @jdumars @jberkus @tpepper 
/hold

### CODE REVIEW

**Comment by calebamiles** [comment]
@justaugustus I would remove the `TODO()` sections since they don't render nicely in markdown, perhaps tack on an additional `TODO.md` or something in the PR. I would also suggest exporting the features tracking spreadsheet as a CSV for archival in the `release-X.Y` directory as one of the tasks but otherwise looks good to me. Thanks!

**Comment by calebamiles** [comment]
Oh @justaugustus I would also consider adding the release lead to both approvers and reviewers in the `OWNERS` file.

**Comment by justaugustus** [comment]
1.12 Features Shadows, please review the Features Handbook and provide feedback when you have a chance.
/cc @kacole2 @robertsandoval @rajendar38

**Comment by justaugustus** [comment]
@tpepper -- Addressed your notes about `Limitations`. Can you re-`/lgtm` when you have a chance?

**Comment by justaugustus** [comment]
I'll keep this held until our wonderful Features Lead Emeritus, @idvoretskyi, has a chance to review.

**Comment by idvoretskyi** [comment]
@justaugustus will do by EOD. Thanks!

**Comment by justaugustus** [comment]
Thanks Ihor!

File: release-team/role-handbooks/features/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- Collate the major themes of the release, including but not limited to:
+  - new features
+  - long-awaited features
+  - features moving into GA
+  - feature deprecations
+  - notable changes to existing behaviors
+- Assist in Communications activities (in conjunction with the Communications Lead & the CNCF Communications team):
+  - Draft and / or review https://kubernetes.io/blog/ release announcement post, leveraging the themes collected across the release cycle e.g., [1.11 Announcement][1.11-announcement]
+  - Engage with media analysts during the embargo period to discuss the release themes
+  - CNCF Announcement webinar
```
Comment (idvoretskyi) [inline release-team/role-handbooks/features/README.md]: I'd call it "CNCF Release webinar" or similar.

File: release-team/role-handbooks/features/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  - Identify potential contributors for the “5 Days of Kubernetes” blog series
+  - Identify candidates to assume the Features Lead role (according to the [Release Team selection process][rt-selection]) in the following release cycle
+
+## General Requirements
+
+In addition to the [requirements detailed for all Release Team members][rt-requirements], becoming a Features Lead has additional gates:
+  - MUST have served on the Release Team in a previous capacity, ideally as a Features Shadow
+  - MUST be a member of the SIG Release Google Group
```
Comment (idvoretskyi) [inline release-team/role-handbooks/features/README.md]: Link, please.

File: release-team/role-handbooks/features/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  - MUST be a member of the SIG PM Google Group
```
Comment (idvoretskyi) [inline release-team/role-handbooks/features/README.md]: Link please.

File: release-team/role-handbooks/features/README.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  - MUST be a member of the Kubernetes Milestone Burndown Google Group
```
Comment (idvoretskyi) [inline release-team/role-handbooks/features/README.md]: Link, please.

File: release-team/role-handbooks/features/README.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+### Milestone Activities + Timing
+- Link items from succession section
+- (optional) Watch the kubernetes/features repo to get notified on all issues
+- Ensure that Feature Tracking sheet for the new release was created by the previous Features Lead and populated with the features removed from the previous milestone
+- Features Tracking sheet is shortlinked with the pattern `k8sxyy-features` e.g., http://bit.ly/k8s112-features
+- Ensure the Features Tracking sheet has the appropriate permissions:
+ownership transferred to you
+  - anyone who has the link can comment
+  - Edit access
```
Comment (idvoretskyi) [inline release-team/role-handbooks/features/README.md]: Let's not publish the emails here explicitly (to lower the chance that they'll be spammed).

**Comment by justaugustus** [comment]
@idvoretskyi / @tpepper -- I've addressed all of the comments, so this is ready for an `/lgtm`! :)

/hold cancel

**Pull request merged**